src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java

```diff
@@ -98,13 +102,79 @@ public void editOverlay() {
 		}, () -> NotEnoughUpdates.INSTANCE.openGui = savedGui));
 	}
 
+	@Override
+	public void saveNow() {
+		NotEnoughUpdates.INSTANCE.saveConfig();
+	}
+
+	private Social social(String name, String iconName, String link) {
+		return new Social() {
+			@Override
+			public void onClick() {
+				Utils.openUrl(link);
+			}
+
+			@Override
+			public List<String> getTooltip() {
+				return Arrays.asList(name, "§7Open " + link);
+			}
+
+			@Override
+			public ResourceLocation getIcon() {
+				return new ResourceLocation("notenoughupdates:social/" + iconName + ".png");
+			}
+		};
+	}
+
+	@Override
+	public List<Social> getSocials() {
+		return Arrays.asList(
+			social(
+				"Twitch",
+				"twitch",
+				"https://twitch.tv/moulberry2"
+			),
+			social(
+				"Patreon",
+				"patreon",
+				"https://patreon.com/moulberry"
+			),
+			social(
+				"YouTube",
+				"youtube",
+				"https://www.youtube.com/channel/UCPh-OKmRSS3IQi9p6YppLcw"
+			),
+			social(
+				"Twitter",
+				"twitter",
+				"https://twitter.com/moulberry/"
+			),
+			social(
+				"GitHub",
+				"github",
+				"https://github.com/NotEnoughUpdates/NotEnoughUpdates"
+			),
+			social(
+				"Discord",
+				"discord",
+				"https://discord.gg/moulberry"
+			)
+		);
+	}
+
+	@Override
+	public String getTitle() {
+		return "§7NotEnoughUpdates by §5Moulberry";
```

Comment:
while we are here can we also add the current version?

Reply:
uhh sure.